BeanstalkFarms/Beanstalk
# Use SDK For Swap

Replace the logic in the swap page with the SDK.

- Removed unnecessary logic that the SDK has now
- Fixed UI CI
- Fix UI build command (e.g. used by netlify deploy)
- Bump vite to 3.2.5 to replace the HMR fix from @0xalecks 's original PR https://github.com/BeanstalkFarms/Beanstalk-UI/pull/518. It seemed to fix it.

Manually tested many scenarios such as:
- Swap BEAN/ETH with "forward" quote
- Swap ETH/3CRV (should show "not supported" message
- Reverse quote (type in the "destination" the amount out)
- Click reverse button and get expected results (the button to swap source/destination input)
- Move BEAN from wallet (circulating) to farm balance and back
- Swap BEAN/ETH with amount of BEAN that exceeds farming balance
